src/megatron/bridge/recipes/__init__.py

```
@@ -18,8 +18,14 @@
 This module exposes all recipe configurations from all model families.
 """
 
-from megatron.bridge.diffusion.recipes.flux.flux import *
-from megatron.bridge.diffusion.recipes.wan.wan import *
+# Diffusion recipes require optional dependencies (e.g. diffusers). Keep
+# non-diffusion training paths (LLM/VLM) usable when those deps are absent.
+try:
+    from megatron.bridge.diffusion.recipes.flux.flux import *
```

Flake8 (7.3.0)

[error] 'megatron.bridge.diffusion.recipes.flux.flux.*' imported but unused

(F401)